docs/data-sources/account_members.md

```diff
@@ -17,7 +17,14 @@ Use this data source to obtain user or actor IDs to manage Workspace Access.
 ## Example Usage
 
 ```terraform
-# Query all Members in Account
+terraform {
+  required_providers {
+    prefect = {
+      source = "prefecthq/prefect"
+    }
+  }
+}
+
```

Comment:
The provider specification is implied, looks like we don't specify this in any of the other data source docs:

```
~/code/github.com/prefecthq/terraform-provider-prefect/main
$ rg 'required_providers' docs/data-sources

$
```

We can just remove it - what do you think?